content/applications/hr/fleet/new_vehicle.rst

```diff
@@ -1,33 +1,46 @@
-===================
-Create new vehicles
-===================
+============
+New vehicles
+============
 
 Odoo's *Fleet* app manages all vehicles and the accompanying documentation that comes with vehicle
 maintenance and driver's records.
 
-All vehicles are organized on the main :guilabel:`Fleet` dashboard. Each vehicle has its own
-*vehicle form*, which is displayed as a card in the kanban view, according to it's status. Every
-vehicle form is displayed in its current corresponding kanban stage. The default stages are
-:guilabel:`New Request`, :guilabel:`To Order`, :guilabel:`Ordered`, :guilabel:`Registered`,
-:guilabel:`Downgraded`, :guilabel:`Reserve`, and :guilabel:`Waiting List`.
+All vehicles are organized within the main :guilabel:`Fleet` dashboard. Every vehicle has its own
+*vehicle form*, which is displayed as a card in the Kanban view, according to its status. Each
+vehicle form is displayed in the corresponding Kanban stage based on its status. The default stages
+are :guilabel:`New Request`, :guilabel:`To Order`, :guilabel:`Registered`, and
```

Comment:
I checked with the experts and then tested in a blank DB with only fleet installed- those columns appear because they were added/configured in the runbot and are not actual default columns that appear. I also thought they were default columns, and were in my previous documentation, but I was wrong.